StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts

```diff
@@ -1,55 +1,31 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
-import { Subscription } from 'rxjs';
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
 import { ApiService } from '@shared/services/api.service';
-import { ModalService } from '@shared/services/modal.service';
 import { NodeStatus } from '@shared/models/node-status';
 import { GlobalService } from '@shared/services/global.service';
+import { ElectronService } from 'ngx-electron';
 
 @Component({
   selector: 'app-about',
   templateUrl: './about.component.html',
   styleUrls: ['./about.component.css']
 })
-export class AboutComponent implements OnInit, OnDestroy {
+export class AboutComponent implements OnInit {
+  nodeStatusSubscription$: Observable<NodeStatus>;
+  applicationVersion: string;
 
-  constructor(private globalService: GlobalService, private apiService: ApiService, private genericModalService: ModalService) { }
-
-  private nodeStatusSubscription: Subscription;
-  public clientName: string;
-  public applicationVersion: string;
-  public fullNodeVersion: string;
-  public network: string;
-  public protocolVersion: number;
-  public blockHeight: number;
-  public dataDirectory: string;
+  constructor(
+    private globalService: GlobalService,
+    private apiService: ApiService,
+    private electron: ElectronService
+  ) { }
 
   ngOnInit() {
     this.applicationVersion = this.globalService.getApplicationVersion();
-    this.startSubscriptions();
-  }
-
-  ngOnDestroy() {
-    this.cancelSubscriptions();
-  }
-
-  private startSubscriptions() {
-    this.nodeStatusSubscription = this.apiService.getNodeStatusInterval()
-      .subscribe(
-        (data: NodeStatus) =>  {
-          let statusResponse = data
-          this.clientName = statusResponse.agent;
-          this.fullNodeVersion = statusResponse.version;
-          this.network = statusResponse.network;
-          this.protocolVersion = statusResponse.protocolVersion;
-          this.blockHeight = statusResponse.blockStoreHeight;
-          this.dataDirectory = statusResponse.dataDirectoryPath;
-        }
-      );
+    this.nodeStatusSubscription$ = this.apiService.getNodeStatusInterval();
   }
 
-  private cancelSubscriptions() {
-    if(this.nodeStatusSubscription) {
-      this.nodeStatusSubscription.unsubscribe();
-    }
+  openWalletDirectory(directory: string): void {
+    this.electron.shell.showItemInFolder(directory);
```

Comment:
We also have a mode that allows Core to run in the browser, in that case you won't have access to the electron API.
Please alter the PR the include something like this: https://github.com/stratisproject/StratisCore/blob/master/StratisCore.UI/src/app/shared/services/global.service.ts#L115

Reply:
Good catch, will update.

Reply:
Added the check, no clickable feature if wallet is not served with electron